packages/modules/api-key/src/services/api-key-module-service.ts

```diff
@@ -253,7 +254,8 @@ export default class ApiKeyModuleService<TEntity extends ApiKey = ApiKey>
   }
 
   @InjectManager("baseRepository_")
-  async retrieve(
+  // @ts-expect-error
+  async retrieveApiKey(
```

Comment:
question: Do we need to migrate this service too? It seems we have redundant method implementations no?

Reply:
@olivermrbl they are not redundant in the case of the api keys, since we remove the token before returning

Reply:
Ah sorry, I only checked the method signature. Forgot about the token. My bad